docs/content/preview/develop/common-patterns/timeseries/partitioning-by-time.md

```diff
@@ -37,7 +37,7 @@ CREATE TABLE part_demo (
 ) PARTITION BY RANGE (ts);
 ```
 
-Create partitions for each month.
+Create partitions for each month. Also, create a `DEFAULT` partition that would hold data that does that fall into any of the other partitions.
```

Comment:
"that would hold data that does that fall into any of the other partitions."
that does not fall into?

Reply:
haha. You should have just modified this yourself.